packages/builders/cliff.toml

```
@@ -21,13 +21,16 @@ body = """
 {% for group, commits in commits | group_by(attribute="group") %}
     ## {{ group | upper_first }}
     {% for commit in commits %}
-        - {% if commit.breaking %}\
-            [**breaking**] \
-          {% endif %}\
-            {% if commit.scope %}\
-                **{{commit.scope}}:** \
-            {% endif %}\
+		- {% if commit.scope %}\
+			**{{commit.scope}}:** \
+		  {% endif %}\
             {{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}](https://github.com/discordjs/discord.js/commit/{{ commit.id }}))\
+		{% if commit.breaking %}\
+			\n\n {% raw %}  {% endraw %} ### 💥 Breaking Changes:\n \
```

```suggestion
			\n\n {% raw %}  {% endraw %} ### Breaking Changes:\n \
```

Seeing as I don't have my pc near me, I'll do it this way.